test/unit/lib/plugins/aws/package/compile/events/cognito-user-pool.test.js

```diff
@@ -41,6 +41,26 @@ describe('AwsCompileCognitoUserPoolEvents', () => {
   });
 
   describe('#newCognitoUserPools()', () => {
+    it('should throw when invalid CUP event is given', () => {
+      awsCompileCognitoUserPoolEvents.serverless.service.functions = {
```

Comment:
Let's not use such tests - we are slowly migrating away from them and all new tests should be based on `runServerless` utility

Reply:
Sounds good! I did it this way for file consistency. If wanted, I can update the entire file to use runServerless.

Reply:
I think for not expanding the scope, let's focus on just new tests and if you'd be interested, please do the refactor for all the other tests in a separate PR, it would be much appreciated 🙇